apps/client/src/common/utils/socket.ts

```diff
@@ -196,19 +196,23 @@ export const connectSocket = () => {
         }
         case 'ontime-refetch': {
           // the refetch message signals that the rundown has changed in the server side
-          const { revision, reload } = payload;
-          const currentRevision = ontimeQueryClient.getQueryData<RundownCached>(RUNDOWN)?.revision ?? -1;
-
+          const { reload, target } = payload;
           if (reload) {
             invalidateAllCaches();
-          } else if (revision > currentRevision) {
-            ontimeQueryClient.invalidateQueries({ queryKey: RUNDOWN });
-            ontimeQueryClient.invalidateQueries({ queryKey: CUSTOM_FIELDS });
+          } else if (target === 'RUNDOWN') {
+            const { revision } = payload;
+            const currentRevision = ontimeQueryClient.getQueryData<RundownCached>(RUNDOWN)?.revision ?? -1;
+            if (revision > currentRevision) {
+              ontimeQueryClient.invalidateQueries({ queryKey: RUNDOWN });
+              ontimeQueryClient.invalidateQueries({ queryKey: CUSTOM_FIELDS });
+            }
+          } else if (target === 'REPORT') {
+            ontimeQueryClient.invalidateQueries({ queryKey: REPORT });
           }
           break;
         }
         case 'ontime-flush': {
-          flushBatchUpdates()
+          flushBatchUpdates();
```

Comment:
question: it seems that we dont use the batch updates? maybe we can remove the logic?